src/models/amun.h

```diff
@@ -185,17 +185,27 @@ class Amun : public EncoderDecoder {
   void createAmunConfig(const std::string& name) {
     Config::YamlNode amun;
     auto vocabs = options_->get<std::vector<std::string>>("vocabs");
-    amun["source-vocab"] = vocabs[0];
-    amun["target-vocab"] = vocabs[1];
-    amun["devices"] = options_->get<std::vector<size_t>>("devices");
-    amun["normalize"] = opt<float>("normalize") > 0;
-    amun["beam-size"] = opt<size_t>("beam-size");
-    amun["relative-paths"] = false;
 
-    amun["scorers"]["F0"]["path"] = name;
+    if(options_->get<bool>("relative-paths")) {
+      amun["relative-paths"] = true;
+      auto dirPath = filesystem::Path{name}.parentPath();
+      amun["source-vocab"] = filesystem::relative(filesystem::Path{vocabs[0]}, dirPath).string();
+      amun["target-vocab"] = filesystem::relative(filesystem::Path{vocabs[1]}, dirPath).string();
+      amun["scorers"]["F0"]["path"] = filesystem::Path{name}.filename().string();
+    } else {
+      amun["relative-paths"] = false;
+      amun["source-vocab"] = vocabs[0];
+      amun["target-vocab"] = vocabs[1];
+      amun["scorers"]["F0"]["path"] = name;
+    }
+
     amun["scorers"]["F0"]["type"] = "Nematus";
     amun["weights"]["F0"] = 1.0f;
 
+    amun["devices"] = options_->get<std::vector<size_t>>("devices");
```

Comment:
Let's not set devices. I was thinking about that, but that's not really a model setting.

Reply:
Removed.